### DISCUSSION TITLE
How to change the favicon?

### DISCUSSION THREAD

**Discussion by kissu** [antfu/vitesse#251] [Q&A]
It may be trivial and I maybe missed something, but changing the file `public/favicon.svg` did nothing.

I also removed the PWA module, cleaned my browser in every possible way + tried in incognito (even another browser), nothing so far.

Setting an external URL is working fine
```html
<link rel="icon" href="https://vitejs.dev/logo.svg" type="image/svg+xml">
```
but I'm not able to specify a `href="/favicon.svg"` and have it taken into account

`/public/favicon.svg` is giving me an error (as expected)
> files in the public directory are served at the root path.
Instead of /public/favicon.svg, use /favicon.svg.

**Comment by antfu** [comment]
Maybe the cache on your deploy site? Try changing the filename?

**Reply by kissu** [reply]
I'm trying it locally at first, either in `dev` or after a `build`, I don't see it being changed in any of them.
Also, trying it in a browser in which I never opened the app or in an incognito window, should not have any cache issues.

**Comment by thuongtruong1009** [comment]
Hi @kissu  !
- A simple way is that after you clone this template locally, you just need to replace the default favicon with your own favicon, and should put it at the old path with the same name.
- Hope it helps you!
- If can, you can mark it as an answer for another guy to know about this.
- Thank you!
